api/types/volume/disk_usage.go

```diff
@@ -21,7 +17,7 @@ type VolumesDiskUsage struct {
 
 	// List of volumes.
 	//
-	Items []*volume.Volume `json:"Items,omitempty"`
+	Items []*Volume `json:"Items,omitempty"`
```

Comment:
Orthogonal to this PR, but wondering if we had any reason to use a slice of pointers for these; if not, then perhaps we should change it to a regular slice?